ui/src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,35 @@
+import HEAD from 'next/head';
+import Link from 'next/link';
+import c from './index.module.scss';
+
+const Dashboard = () => {
+  return (
+    <>
+      <HEAD>
+        <title>SOS - Dashboard</title>
```

Comment:
I think we are naming it `SoS` in rest of the app (small `o`). Please follow the same convention.